backend/src/test/resources/tests/sql/form/ABSOLUT/ALIGNMENT/QUARTER YEAR.json

```diff
@@ -37,10 +37,24 @@
   ],
   "content": {
     "tables": [
-      "/shared/vers_stamm.table.json"
+      "/shared/vers_stamm.table.json",
+      "/shared/persons.table.json"
     ],
     "previousQueryResults": [
       "tests/form/EXPORT_FORM/ABSOLUT/SIMPLE/query_results_1.csv"
     ]
+  },
+  "idColumns": {
+    "table": "persons",
+    "ids": [
+      {
+        "name": "ID",
+        "field": "pid",
+        "primaryId": true,
+        "label": {
+          "de": "result"
```

Comment:
Darüber wollte ich noch mit dir reden: Da die `IdColumnConfig` das Label für die `PID` und ggf. auch die `dates` in der `ResultCsv` überschreibt, müsste ich den Spaltennamen in den `expected.csv`'s von result->pid (in diesem Fall) ändern. Da wollte ich aber erstmal nochmal dein Feedback einholen, ob ich da sowieso nochmal irgendwo ran muss.